data/json/mapgen/survivor_forest_camp.json

```diff
@@ -0,0 +1,151 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "survivor_forest_camp" ],
+    "weight": 100,
+    "object": {
+      "fill_ter": "t_grass",
+      "predecessor_mapgen": "forest_thick",
+      "rows": [
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        ",,,,,,,,,,     ,,,,,,,  ",
+        ",,,,,,,,,,     ,,,,,,,  ",
+        ",,,E,,,,,,     ,,E,,,,  ",
+        ",,,,,,,,,,,,,,,,M,,,,,,,",
+        ",,,,,,,,,,,,,,,,B,,,,,,,",
+        ",,,,,,,,,,,A@,,,C,,,,,,,",
+        ",,,,,,,,,,,,O,,,S,,,,,,,",
+        ",,,,,,,,,,,,@,,,,,,,,,,,",
+        ",,,,,,,,,,,,,,,,,,,,,,,,",
+        "       ,,,E,,,,,,,      ",
+        "       ,,,,,,,,,,,      ",
+        "       ,,,,,,,,,,,      ",
+        "       ,,,,,,,,,,,      ",
+        "       ,,,,,,,,,,,      ",
+        "       ,,,,,,,,,,,      ",
+        "       ,,,,@,,,,,,      ",
+        "       ~~~~~~~~~        ",
+        "      ~~~~~~~~~~~       ",
+        "     ~~~~~~~~~~~~~      ",
+        "      ~~~~~~~~~~~       "
+      ],
+      "flags": [ "ERASE_ALL_BEFORE_PLACING_TERRAIN" ],
+      "nested": {
+        "E": { "chunks": [ [ "leather_tent_5x5_S", 100 ] ] },
+        "A": { "chunks": [ [ "survivor_corpse", 25 ], [ "survivor_creature", 75 ] ] }
+      },
+      "terrain": {
+        ",": [ "t_grass" ],
+        "~": "t_water_dp",
+        "O": "t_dirt",
+        "@": "t_grass",
+        "E": "t_grass",
+        "S": "t_dirt",
+        "C": "t_dirt",
+        "B": "t_grass",
+        "A": "t_grass",
+        "M": "t_grass"
+      },
```

Comment:
You'll need to regionalize your terrains here (`t_dirt` --> `t_region_soil`, `t_grass` --> `t_region_groundcover`)